eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -402,33 +427,21 @@ jobs:
 
         buildArgs="$(additionalBuildArgs) $customBuildArgs $extraBuildProperties"
 
-        # Only use Docker when a container is specified
-        if [[ -n "${{ parameters.container }}" ]]; then
-          # Allows Arcade to have access to the commit for the build, pass it through to the container
-          customEnvVars="$customEnvVars BUILD_SOURCEVERSION=$BUILD_SOURCEVERSION"
-
-          customEnvVars="$customEnvVars BUILD_BUILDNUMBER=$(Build.BuildNumber)"
-
-          customDockerRunArgs=""
-          for envVar in $customEnvVars; do
-            customDockerRunArgs="$customDockerRunArgs -e $envVar"
-          done
-
-          if [[ '${{ parameters.runOnline }}' == 'False' ]]; then
-            customDockerRunArgs="$customDockerRunArgs --network none"
-          fi
-
-          docker run --rm -v "$(sourcesPath):/vmr" -w /vmr $customDockerRunArgs ${{ parameters.container }} ./build.sh $buildArgs
-        else
-          for envVar in $customEnvVars; do
-            customEnvVarsWithBashSyntax="$customEnvVarsWithBashSyntax export $envVar;"
-          done
-          cd $(sourcesPath)
-          eval $customEnvVarsWithBashSyntax
-          ./build.sh $buildArgs
-        fi
+        for envVar in $customEnvVars; do
+          customEnvVarsWithBashSyntax="$customEnvVarsWithBashSyntax export $envVar;"
+        done
+        cd $(sourcesPath)
+        eval $customEnvVarsWithBashSyntax
+        ./build.sh $buildArgs
       displayName: Build
 
+    - ${{ if ne(parameters.runOnline, 'True') }}:
+      - script: |
+          set -ex
+          sudo ip link set eth0 up
```

Comment:
After enabling the network, the offline build fails while running the tests:

```
/__w/1/s/test/TestUtilities/TestUtilities.csproj : error NU1301: Unable to load the service index for source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json. [/__w/1/s/test/tests.proj]
```

I'm investigating

Reply:
I the connection has been reestablished, but DNS resolution is not working.

Reply:
Using `sudo unshare -n` appears to work.